src/ui_fsmenu/singleplayersetupbox.cc

```diff
@@ -140,26 +140,25 @@ SinglePlayerSetupBox::SinglePlayerSetupBox(UI::Panel* const parent,
             g_style_manager->font_style(UI::FontStyle::kFsGameSetupHeadings)) {
 	add(&title_, Resizing::kAlign, UI::Align::kCenter);
 	add_space(3 * padding);
-	add(&scrollableBox_, Resizing::kAlign, UI::Align::kCenter);
+	add(&scrollableBox_, Resizing::kExpandBoth);
```

Comment:
I only see this now… but please use under_scores instead of camelCase in variable names.

Reply:
done